librocksdb_sys/crocksdb/c.cc

```diff
@@ -4109,9 +4110,14 @@ struct ExternalSstFileModifier {
     sst_file_reader.reset(new RandomAccessFileReader(std::move(sst_file), file_));
 
     // Get Table Reader
-    status = cfd_->ioptions()->table_factory->NewTableReader(
-        TableReaderOptions(*cfd_->ioptions(), env_options_,
-                           cfd_->internal_comparator()),
+    ColumnFamilyDescriptor desc;
+    handle_->GetDescriptor(&desc);
+    auto cfd = reinterpret_cast<ColumnFamilyHandleImpl*>(handle_)->cfd();
+    auto ioptions = *cfd->ioptions();
+    status = ioptions.table_factory->NewTableReader(
```

Comment:
The `NewTableReader` arguments are changed, so I need to refactor this function a little bit.